app/cdap/components/FetchIconsFromWidget/types.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export interface IConnectorArray {
+  name: string;
+  typ?: string;
+  category?: string;
+  description?: string;
+  artifact?: {
+    name: string;
```

Comment:
make this separate interface and call here
interface IArtifact {
    name: string;
    version: string;
    scope: string;
  }

then
artifact: IArtifact

Reply:
Done